openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -35,24 +35,51 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                                 >
                                     <div class="subsection-text">
                                         ## Subsection title
-                                        <span class="subsection-title">${ subsection['display_name'] }</span>
-
+                                        <span class="subsection-title">
+                                            % if subsection['id'] in milestones:
+                                                % if milestones[subsection['id']]['completed_prereqs']:
+                                                    <span class="menu-icon icon fa fa-unlock"
+                                                        aria-hidden="true">
+                                                    </span>
+                                                % else:
+                                                    <span class="menu-icon icon fa fa-lock" 
+                                                    aria-hidden="true">
+                                                    </span>
+                                                % endif
+                                            % endif
+                                            <span class="subsection-title-name">
+                                                ${ subsection['display_name'] }
+                                            </span>
+                                            % if subsection['id'] in milestones: 
+                                                % if milestones[subsection['id']]['completed_prereqs']:
+                                                    <span class="sr">&nbsp;${_("Unlocked")}</span>
+                                                % else:
+                                                    <span class="details">
+                                                        <span class="subtitle">
+                                                            <span class="subtitle-name">
```

Comment:
Can these 3 spans be combined into one?
```
<span class="details subtitle subtitle-name">
${ _("(Prerequisite required)") }
</span>
```

Reply:
That's a good idea - I'll compress those down